habushu-maven-plugin/src/main/java/org/technologybrewery/habushu/InstallDependenciesMojo.java

```diff
@@ -6,7 +6,6 @@
 import com.vdurmont.semver4j.Semver;
 import org.apache.commons.collections4.CollectionUtils;
 import org.apache.commons.collections4.MapUtils;
```

Comment:
I (Informational): I cleaned up some sonarlint items as a means to test invalidating the cache.  So conveying those small tweaks into the PR as well.